### ISSUE TITLE
No CPU tracking on Colab, CODECARBON : No CPU tracking mode found. Falling back on CPU constant mode.

### ISSUE THREAD

**Issue by Hevia** [mlco2/codecarbon#282]
* CodeCarbon version: 1.2.0
* Python version: 3.7.2
* Operating System: Linux (whatever distro Google Colab uses)

### Description
When trying to track emissions of my model running on Google Colab I get these errors:
```
CODECARBON : No CPU tracking mode found. Falling back on CPU constant mode.
CODECARBON : Failed to match CPU TDP constant. Falling back on a global constant.
```

### What I Did
I followed the instructions from this issue: https://github.com/mlco2/codecarbon/issues/244 it seems like it was successful, no error running any commands. 

Here is my `sysfs.conf` file:
![image](https://user-images.githubusercontent.com/15069763/153690388-05c1b84d-eadc-42fb-95e1-694fb4c917e4.png)

I also ran this command to see what CPU I have:
```
>!lscpu | egrep 'Model name|Socket|Thread|NUMA|CPU\(s\)'
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  2
Socket(s):           1
NUMA node(s):        1
Model name:          Intel(R) Xeon(R) CPU @ 2.20GHz
NUMA node0 CPU(s):   0-3
```

I also noticed from this pull request: https://github.com/mlco2/codecarbon/pull/253/files the default path of `/sys/class/powercap/intel-rapl` I don't see that path in the filesys.

Since I could not fully restart my colab instance I ran the `sudo chmod -R a+r /sys/class/powercap/intel-rapl` and got the error: `chmod: cannot access '/sys/class/powercap/intel-rapl': No such file or directory`.  Is there an alternative place this can be found? Not remarkably familiar with this 😅

Thanks for any help! I can try and see if I encounter the same issue on my local machine but would prefer to stick to Colab if possible.

**Comment by benoit-cty** [comment]
Hello,

Thanks for reporting, unfortunately Google Colab do not expose CPU power to user, so you could not use CodeCarbon on Colab for CPU.
FYI it works on Colab with GPU.

**Comment by ashokkushh** [comment]
`[codecarbon INFO @ 21:18:08] [setup] RAM Tracking...
[codecarbon INFO @ 21:18:08] [setup] GPU Tracking...
[codecarbon INFO @ 21:18:08] No GPU found.
[codecarbon INFO @ 21:18:08] [setup] CPU Tracking...
[codecarbon WARNING @ 21:18:08] No CPU tracking mode found. Falling back on CPU constant mode.
[codecarbon INFO @ 21:18:11] CPU Model on constant consumption mode: Intel(R) Core(TM) i3-5005U CPU @ 2.00GHz
[codecarbon INFO @ 21:18:11] >>> Tracker's metadata:
[codecarbon INFO @ 21:18:11]   Platform system: Windows-10-10.0.19045-SP0
[codecarbon INFO @ 21:18:11]   Python version: 3.11.1
[codecarbon INFO @ 21:18:11]   CodeCarbon version: 2.2.4
[codecarbon INFO @ 21:18:11]   Available RAM : 7.923 GB
[codecarbon INFO @ 21:18:11]   CPU count: 4
[codecarbon INFO @ 21:18:11]   CPU model: Intel(R) Core(TM) i3-5005U CPU @ 2.00GHz
[codecarbon INFO @ 21:18:11]   GPU count: None
[codecarbon INFO @ 21:18:11]   GPU model: None`

getting same issue!!
please Help

**Comment by benoit-cty** [comment]
Hello, we have a [PR open](https://github.com/mlco2/codecarbon/pull/316) to estimate CPU power from CPU load, you can test it on Colab if you know how to do it.
